demosplan/DemosPlanCoreBundle/ResourceTypes/GisLayerResourceType.php

```diff
@@ -84,6 +84,11 @@ public function isAvailable(): bool
         return true;
     }
 
+    public function isDeleteAllowed(): bool
```

Comment:
Lets add the permission area_map_participation_area as it is done with the isGetAllowed, and isListAllowed, in this way, the deletion is also protected

Reply:
done in [5250f95](https://github.com/demos-europe/demosplan-core/pull/3347/commits/5250f95a3e681ed932017b8b3b13dcb706a4156f)